docs/getting-started.rst

```diff
@@ -94,80 +92,78 @@ We can run it, and see the traces print to your console:
 
 Each span typically represents a single operation or unit of work.
 Spans can be nested, and have a parent-child relationship with other spans.
-While a given span is active, newly-created spans will inherit the active span's trace ID, options, and other attributes of its context.
-A span without a parent is called the "root span", and a trace is comprised of one root span and its descendants.
+While a given span is active, newly-created spans inherit the active span's trace ID, options, and other attributes of its context.
+A span without a parent is called the root span, and a trace is comprised of one root span and its descendants.
 
-In the example above, the OpenTelemetry Python library creates one trace containing three spans and prints it to STDOUT.
+In this example, the OpenTelemetry Python library creates one trace containing three spans and prints it to STDOUT.
 
 Configure exporters to emit spans elsewhere
 -------------------------------------------
 
-The example above does emit information about all spans, but the output is a bit hard to read.
-In common cases, you would instead *export* this data to an application performance monitoring backend, to be visualized and queried.
-It is also common to aggregate span and trace information from multiple services into a single database, so that actions that require multiple services can still all be visualized together.
-
-This concept is known as distributed tracing. One such distributed tracing backend is known as Jaeger.
+The previous example does emit information about all spans, but the output is a bit hard to read.
+In most cases, you can instead *export* this data to an application performance monitoring backend to be visualized and queried.
+It's also common to aggregate span and trace information from multiple services into a single database, so that actions requiring multiple services can still all be visualized together.
 
-The Jaeger project provides an all-in-one docker container that provides a UI, database, and consumer. Let's bring
+This concept of aggregating span and trace information is known as distributed tracing. One such distributed tracing backend is known as Jaeger. The Jaeger project provides an all-in-one Docker container with a UI, database, and consumer. Let's bring
```

Comment:
The "let's bring it up now" doesn't really flow with the 2nd person edits in the rest of the doc. Maybe remove it, or something like "You can execute the code below to do this".

Reply:
Fixed.